source/administration/devops-command-center.rst

```diff
@@ -63,13 +63,21 @@ Mattermost Incident Management includes built-in slash commands:
 Adding slash commands to tasks
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Slash commands can be added to tasks to initiate actions as part of your incident response playbook.
+Slash commands can be added to tasks to initiate actions as part of your playbook.
 
 Here are some examples:
 
 - Add a task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 
 - One of your tasks may require the header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.
 
+Generating test data
+^^^^^^^^^^^^^^^^^^^^
+
+You can use the test commands to create incidents that are populated with random data. These incidents are listed in the incident insights page.
```

Comment:
Is it "incident insight page" or "incident insights page"? Ensure that all references, including Glossary entries below, reference the page consistently.